src/test/kotlin/LoxoneAT.kt

```diff
@@ -0,0 +1,188 @@
+package cz.smarteon.loxone
+
+import cz.smarteon.loxone.app.SwitchControl
+import cz.smarteon.loxone.message.JsonValue
+import cz.smarteon.loxone.message.LoxoneMessage
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.MethodOrderer
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestMethodOrder
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable
+import strikt.api.expectCatching
+import strikt.api.expectThat
+import strikt.assertions.hasSize
+import strikt.assertions.isA
+import strikt.assertions.isEqualTo
+import strikt.assertions.isNotEqualTo
+import strikt.assertions.isSuccess
+import strikt.assertions.isTrue
+import java.security.Security
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import kotlin.collections.LinkedHashMap
+import kotlin.collections.first
+import kotlin.collections.set
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+@EnabledIfEnvironmentVariable(named = LoxoneAT.LOX_ADDRESS, matches = ".*")
```

Comment:
add some meaningfull patterns - for example empty string should not match, no need to match all the possible shapes of address on the other hand